ci-operator/step-registry/openstack-k8s-operators/gather/openstack-k8s-operators-gather-commands.sh

```diff
@@ -1,21 +1,126 @@
 #!/usr/bin/env bash
 
 set -x
+set +eu
+
+BASE_DIR=${HOME:-"/alabama"}
+MUST_GATHER_TIMEOUT=${MUST_GATHER_TIMEOUT:-"5m"}
+NS_OPERATORS=${NS_OPERATORS:-"openstack"}
+NS_SERVICES=${NS_SERVICES:-"openstack"}
+
+# Creates directory if does not exists
+#  Parameters:
+#  $1: directory to be created
+function create_out_dir {
+  local OUT_DIR=${1}
+  if [ ! -d "./${OUT_DIR}" ]; then
+    mkdir -p ${OUT_DIR}
+  fi
+}
+
+# Generates a debug command tp get pods logs and description. To be used with "xargs -I {}".
+#  Parameters:
+#  $1: directory to store output logs (optional)
+function gen_pods_debug_cmd {
+  local cmd
+  if [ -z "${1}" ]; then
+    cmd=("echo 'Logs for pod: {}'; oc describe pod {} > {}-describe.log; oc logs --all-containers=true {} >> {}-containers.log")
```

Comment:
nit: Can we instruct `oc logs` to somehow separate logs from different containers in the same Pod in the output. I only found `--prefix`, that would include the name of the container into the output. For me a single empty line or any other clear separator would be enough between the logs though. If that is not possible then I would like to have --prefix added to the command.

Reply:
Good idea, I vote for using `--prefix` for now. We may want to improve that in a next PR if needed.